styles.css

```diff
@@ -0,0 +1,53 @@
+
+body {
+  margin: 0%;
+}
+
+.nav {
+  display: flex;
+  justify-content: space-between;
+}
+
+.hamb {
+  margin-top: 19px;
+  margin-right: 16px;
+}
+
+.mix1 {
+  margin-top: 32.5px;
+}
+
+h1 {
+  padding-top: 30px;
+  font-family: 'Playfair Display', serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 50px;
+  line-height: 50px;
+  color: #1a2236;
+}
+
+.firstp {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 21px;
+  letter-spacing: -0.01em;
+}
```

Comment:
Please adjust your text margin to be as similar as possible to the one in the design. Keep in mind that this project's goal is to improve your ability to follow design guidelines (imagine Microverse as a client requesting a website) 😉.
| Figma design  | Your design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/42799579/198191953-03e617f5-7be4-471b-a89b-8179c8c9af50.png)  | ![image](https://user-images.githubusercontent.com/42799579/198191859-2322e0f4-0536-4bcd-a0c8-a7764591d675.png)  |
> Kudos for using the correct font-family 😄.